src/application/wizard/service.ts

```diff
@@ -192,6 +183,24 @@ export class WizardService {
         if (!allowed) {
             throw new Error('User not allowed to update submission');
         }
-        return await this.submissionService.changeCoverLetter(submissionId, coverLetter);
+
+        await this.submissionService.changeCoverLetter(submissionId, coverLetter);
+
+        return this.getFullSubmission(submissionId);
+    }
+
+    /**
+     * Return a full submission
+     *
+     * @param submissionId
+     */
+    private async getFullSubmission(submissionId: SubmissionId): Promise<Submission> {
+        const submission = await this.submissionService.get(submissionId);
```

Comment:
Maybe its worth adding a comment that teams needs to go onto here to eventually?

Reply:
true will add it